src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
```

Comment:
The `alt` attribute of the `img` tag should describe the image content. 'Moyo logo' is not very descriptive. It's better to describe what is in the image. For example, 'Company Logo Moyo'.